app/controllers/admin/lead_providers/lead_provider_users_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  module LeadProviders
+    class LeadProviderUsersController < Admin::BaseController
+      skip_after_action :verify_policy_scoped
+      before_action :load_lead_provider_user
+
+      def edit
+        authorize User
+      end
+
+      def update
+        authorize User
```

Comment:
Might also be good to authorize LeadProviderProfile?

Reply:
Or even better, authorize @lead_provider_user. That way in future, we can write clever policies that check which profiles a user has before permitting editing